frontend/tests/jest/__mocks__/products.ts

```diff
@@ -1,13 +1,14 @@
-import { CurrencyCode, FindProductQuery } from '@lib/shopify-storefront-sdk';
-import { MenuItemType } from '@models/menu';
-import { ProductSearchHit } from '@models/product-list';
-import { ProductReview } from '@models/product/reviews';
+import { CurrencyCode } from '@lib/shopify-storefront-sdk';
+import type { FindProductQuery } from '@lib/shopify-storefront-sdk';
+import { MenuItemType } from '@ifixit/ui/menu';
```

Comment:
```suggestion
import { MenuItemType } from '@ifixit/ui';
```

Don't know if it makes a difference.

Reply:
We can't do this yet ([issue](https://github.com/iFixit/react-commerce/issues/1400#issue-1595575920)):
> The cause of this issue was the `index` file of the package we were trying to import, `@ifixit/ui`, in the mocked products file[^2], also included Typescript XML, `tsx`, files. This became a problem with Playwright since we use this `products.ts` file in one of our tests[^3]. Playwright tries to compile all the imported files defined in the `index` file of `@ifixit/ui` which included the `tsx` files[^4]. However, Playwright does not know how to parse `tsx` files so it throws an error. 

> The workaround for this was to explicitly import the `ts` file needed in the `products.ts` file - i.e. `@ifixit/ui/menu` instead of `@ifixit/ui` [^5]